triple-store/triple-store-api/src/main/java/com/powsybl/triplestore/api/PropertyBag.java

```diff
@@ -173,9 +174,10 @@ public final boolean equals(Object obj) {
 
     public boolean isResource(String name) {
         // TODO do not rely on property name, use metadata or answer based on value?
-        return name.equals("TopologicalNode") || name.equals("Terminal")
-                || name.equals("ShuntCompensator") || name.equals("TapChanger")
-                || resourceNames.contains(name);
+        List<String> list = Stream.of("TopologicalNode", "Terminal", "ShuntCompensator",
+            "TapChanger", "ConductingEquipment", "Model.DependentOn", "TopologicalNodes",
+            "AngleRefTopologicalNode").collect(Collectors.toList());
```

Comment:
Replace `Stream.of` by `Arrays.asList`. This list should be static and final ?

Reply:
Done